bandit/core/issue.py

```diff
@@ -31,6 +31,7 @@ class Cwe:
     IMPROPER_CHECK_OF_EXCEPT_COND = 703
     INCORRECT_PERMISSION_ASSIGNMENT = 732
     INAPPROPRIATE_ENCODING_FOR_OUTPUT_CONTEXT = 838
+    DOWNLOAD_OF_CODE_WITHOUT_INTEGRITY_CHECK = 494
```

Comment:
Please keep these sorted by number. Makes it easier to detect whether we already have a constant for that CWE